### ISSUE TITLE
setNumber with Tuvalu phone number does not select flag, other countries work fine

### ISSUE THREAD

**Issue by fishnrod** [jackocnr/intl-tel-input#583]
While other countries work fine, when I try to do a setNumber with a valid number from Tuvalu (Country code 688), the Tuvalu flag is not set.  

Using: International Telephone Input v10.0.0

### Steps to reproduce
1.  Create an instance of the widget like this:
phone.intlTelInput({
	separateDialCode:"true",
	utilsScript:"/etc/designs/mt/widgets/jqlibs/intl-tel-input/js/utils.js",
	preferredCountries:[],
	initialCountry: "us",
});

2.  Issue the setNumber for a number from Tuvalu
phone.intlTelInput("setNumber","+688901234")  

### Expected behaviour
The flag should be the Tuvalu flag, the phone number should be 901234

### Actual behaviour
The flag is the US flag, the phone number is correct at 901234

The other countries I tested all worked correctly.  However, my testing was not exhaustive.  I thought I would log this issue in case there was something deeper than just a problem with Tuvalu.

Thanks,
Rod

**Comment by jackocnr** [comment]
Hmm it seems to work for me: http://codepen.io/jackocnr/pen/LyxQmq

Perhaps try upgrading to the latest version. And you have initialCountry:us - is that necessary in this case?

**Comment by fishnrod** [comment]
Clearly you are correct.  I even updated the codepen to point to our version of the utils, and it still works.  The problem must be on my end.  Sorry for the trouble.  Thanks for responding so quickly.

And, yes, the US was necessary, as we set the default country flag depending on which country on our site you are browsing in.  I had suspected that this was somehow involved in the problem, but clearly it isn't.

Thanks for the help and the great tool!
